sdk/trace/trace_test.go

```diff
@@ -128,11 +128,11 @@ func TestSetName(t *testing.T) {
 	}
 }
 
-func TestRecordingIsOff(t *testing.T) {
+func TestRecordingIsOn(t *testing.T) {
 	tp, _ := NewProvider()
 	_, span := tp.Tracer("Recording off").Start(context.Background(), "StartSpan")
```

Comment:
This tests seems it was broken from the start :joy:. It "worked" because the old sample chance was small enough that this have always be off. Thanks for fixing this.

Could you also change the strings on this test? `"Recording off"` and `"new span is recording events"`